install

```diff
@@ -89,14 +89,29 @@ pause
 
 say-loud "Setting ZSH as the default shell"
 say "You might be asked for your login password..."
-# shellcheck disable=SC2016
-run 'echo "$(brew --prefix)/bin/zsh" | sudo tee -a /etc/shells'
+if ! sudo grep "$(brew --prefix)/bin/zsh" /etc/shells; then
+  # shellcheck disable=SC2016
+  run 'echo "$(brew --prefix)/bin/zsh" | sudo tee -a /etc/shells'
+fi
 
 # shellcheck disable=SC2016
 run 'chsh -s "$(brew --prefix)/bin/zsh" "$USER"'
 
-say-loud "Creating default ZSH config..."
-run 'cp src/zshrc ~/.zshrc'
+DEFAULT_ZSHRC=
+if [ -f ~/.zshrc ]; then
+  if ask_Yn "Do you want to overwrite your existing ZSH config?"; then
```

Comment:
Would this make more sense as:
if zshrc:
   set default flag
else:
   if overwrite:
      backup
      set default flag

if default flag:
   install default script
   
(it's rather marginal)

Reply:
It would be less repetitive at the cost of being probably a bit less clear due to breaking conceptually connected logic apart, imo.

Reply:
Consider me partially persuaded.